src/test/java/com/github/dockerjava/core/command/ListNetworksCmdImplTest.java

```diff
@@ -45,11 +46,23 @@ public void listNetworks() throws DockerException {
 
         List<Network> networks = dockerClient.listNetworksCmd().exec();
 
-        Network network = networks.get(0);
```

Comment:
I don't think the order the networks are returned is deterministic. In my environment, the "none" network is returned first. I updated the test to ensure the bridge network exists in the list, but not necessarily first.